src/metric.ts

```diff
@@ -0,0 +1,9 @@
+export default class Metric {
```

Comment:
A better name for this class is **PerformanceEntry**, because we are storing the `name` and `duration` coming from the [Performance Observer](https://w3c.github.io/performance-timeline/#dom-performanceentry) and [Performance Mark](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark) APIs.

Reply:
Done.

Reply:
Hm, there's a problem with this. It conflicts with an existing PerformanceEntry type in lib.dom.ts. I've namespaced it for now. Got another suggested name or is namespacing fine?